humility-core/src/hubris.rs

```diff
@@ -3933,11 +3933,8 @@ impl HubrisObjectLoader {
                 continue;
             }
 
-            let name = match elf.strtab.get(sym.st_name) {
-                Some(n) => n?,
-                None => {
-                    bail!("bad symbol in {}: {}", object, sym.st_name);
-                }
+            let Some(name) = elf.strtab.get_at(sym.st_name) else {
+                bail!("bad symbol in {}: {}", object, sym.st_name);
```

Comment:
nit, take it or leave it:
```suggestion
                bail!("bad symbol in {object}: {}",, sym.st_name);
```